frontend/src/tests/assignment-tests.js

```diff
@@ -1,10 +1,125 @@
-import { it } from "./utils";
+import PropTypes from "prop-types";
+import {
+    it,
+    expect,
+    beforeAll,
+    checkPropTypes,
+    assignmentPropTypes
+} from "./utils";
+import { databaseSeeder } from "./setup";
 
-// TODO: Remove eslint disable
-// eslint-disable-next-line
 export function assignmentsTests({ apiGET, apiPOST }) {
-    it.todo("get assignments for session");
-    it.todo("get assignments for position");
-    it.todo("create and delete assignment for position");
-    it.todo("update assignment dates/note/offer_override_pdf");
+    const session = databaseSeeder.seededData.session;
+    const position = databaseSeeder.seededData.position;
+    const assignment = databaseSeeder.seededData.assignment;
+
+    beforeAll(async () => {
+        await apiPOST("/debug/restore_snapshot");
+    }, 30000);
+
+    it("get assignments for session", async () => {
+        const resp = await apiGET(`/admin/sessions/${session.id}/assignments`);
```

Comment:
maybe we can compare the response with the data in `databaseSeeder.seededData`

Reply:
I believe that is what she's doing. In like 25 she compares the response payload to seededData.assignment.

Or are you suggesting we also compare the response to the seededData response and not the explicit value of ``status: success``?

Reply:
What is happening here after the get response is: 

1. check the response status and make sure the api call succeed. (line 23)
2. compare the payload with seeded data (line 25)
3. check if payload matches the proptype (line 27)

Reply:
oops, I was blind, my bad